saleor/webhook/tests/test_webhook_payloads.py

```diff
@@ -82,3 +88,18 @@ def test_order_lines_have_all_required_fields(order, order_line_with_one_allocat
         ),
         "tax_rate": str(line.tax_rate.quantize(Decimal("0.0001"))),
     }
+
+
+def test_generate_product_variant_payload(product_with_variant_with_two_attributes):
```

Comment:
We should add test `test_generate_product_variant_payload_for_deleted_variant`

Reply:
What's the point here? It's exactly the same ATM.